test/e2e/0_configuration.sh

```diff
@@ -1,25 +1,27 @@
-#!/bin/bash
-set -e
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
 shopt -s expand_aliases
 
-alias hardhat="node packages/ethereum/node_modules/.bin/hardhat"
+alias hardhat="yarn hardhat"
```

Comment:
This one is also mistaken, as `yarn hardhat` isn't a script on the root package, but only available within `packages/ethereum`

Reply:
all of these binaries are available in the top-level as well